src/app/components/comments/comments.component.scss

```diff
@@ -2,6 +2,9 @@
 
 .comments-container {
   background-color: $default-bg-color;
+  max-width: 600px;
```

Comment:
_suggestion_: maybe extract this to a named SASS variable, since its used to two places (line 52 in this file also defines `max-width: 600px`)